packages/scroller/src/vaadin-scroller.d.ts

```diff
@@ -36,6 +36,16 @@ declare class Scroller extends FocusMixin(ThemableMixin(ElementMixin(ControllerM
    * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
    */
   tabindex: number;
+
+  /**
+   * Scrolls the textarea to the start if it has a vertical or horizontal scrollbar.
```

Comment:
The description doesn't state that the function only has an effect if `scrollDirection` is set. Shouldn't there be a way to scroll programmatically even if the direction is undefined?

Reply:
> Shouldn't there be a way to scroll programmatically even if the direction is undefined?

What I meant is that if the scroller supports scrolling in both directions (like the scroller on the dev page or [this docs example](https://vaadin.com/docs/latest/components/scroller#both)), should there be a way to scroll to end/start in either of them? Has this API been discussed somewhere (internally)?

Reply:
Right. Not in detail, no. I thought about what "end" means when there is overflow in both directions, but based on the original issue and some very brief internal discussion I ended up with this. The original issue really only wanted a way to scroll to the top, but maybe this requires a bit more thought.

I think there will always be bias towards one of the scroll directions, even 2D UIs like a Grid. I don't think "end" should scroll to the diagonally opposite corner. So horizontal scroll direction should support manual vertical scrolling, but the programmatic scroll should only apply to the given scroll.

Reply:
One option could be to have it default to the vertical axis as it is now, but support an optional parameter with the same type as `scrollDirection`, for example `scrollToEnd('horizontal')`. That's something we could also add later if there is demand for it.